gprofiler/gprofiler_types.py

```diff
@@ -56,3 +56,25 @@ def integer_range_check(value_str: str) -> int:
         return value
 
     return integer_range_check
+
+
+def sorted_positive_int_list(s: str) -> List[int]:
+    s = s.strip(" \t[]")
```

Comment:
What's the idea? To handle `[1,2,3]` and also separated with spaces/tabs? Please comment

Reply:
Let's remove thhe ` ` and `\t` from that - these are ignored when parsing a single int (acceptable) but will raise an error if the list was separated by spaces instead of commas for whatever reason:
```
In [3]: int("\t123 ")                                                                                                                                                                                              
Out[3]: 123

In [4]: int("\t123 4")                                                                                                                                                                                             
---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
<ipython-input-4-7cb2b6a777da> in <module>
----> 1 int("\t123 4")

ValueError: invalid literal for int() with base 10: '\t123 4'
```